src/Http/RequestBuilder.php

```diff
@@ -319,6 +319,6 @@ public function build(): Request
             isset($this->payload) ? $this->payload->toStream() : null
         );
 
-        return new Request($message, $this->client ?? $this->config->getHttpClient() ?? new Client());
+        return new Request($message, $this->client ?? $this->config->getHttpClient());
```

Comment:
`Config::getHttpClient()` has been changed to return a default client when one hasn't been configured, so we no longer need to create one here